test/ntp_client_test.rb

```diff
@@ -0,0 +1,487 @@
+require_relative "test_helper"
+
+Yast.import "NtpClient"
+Yast.import "NetworkInterfaces"
+Yast.import "PackageSystem"
+Yast.import "Service"
+
+describe Yast::NtpClient do
+
+  subject { Yast::NtpClient }
+
+  describe "#Read" do
+    before do
+      subject.config_has_been_read = false
+      allow(Yast::NetworkInterfaces).to receive(:Read)
+      allow(subject).to receive(:Abort).and_return(false)
+      allow(Yast::Mode).to receive(:normal).and_return(false)
```

Comment:
You could mock `Yast::Mode.mode` so we you'll get `Yast::Mode.normal`, `Yast::Mode.installation`, etc. for free :) BTW, is the `Mode.install == true` scenario tested?